Form/Type/MediaType.php

```diff
@@ -35,13 +35,19 @@ class MediaType extends AbstractType
     protected $class;
 
     /**
+     * @var bool
+     */
+    private $isSymfony28;
```

Comment:
@Soullivaneuh Do not use `private` properties or do not use `properties`? I'm not sure if you want I remove it or you want I change the encapsulation?

@core23 Looks like @Soullivaneuh doesn't like it. (comment in PR)